pyaedt/hfss.py

```diff
@@ -6117,17 +6181,17 @@ def parse_hdm_file(self, filename):
 
         from pyaedt.sbrplus.hdm_parser import Parser
 
-        if os.path.exists(filename):
-            return Parser(filename).parse_message()
+        if os.path.exists(file_name):
+            return Parser(file_name).parse_message()
         return False
 
-    @pyaedt_function_handler()
-    def get_hdm_plotter(self, filename=None):
+    @pyaedt_function_handler(filename="file_name")
+    def get_hdm_plotter(self, file_name=None):
         """Get the ``HDMPlotter``.
```

Comment:
```suggestion
        """Get the  HDM plotter``.
```
Try to avoid code entities in brief descriptions.